pages/available-tokens/color-tokens.en.mdx

```diff
@@ -2,46 +2,47 @@ import Callout from 'nextra-theme-docs/callout'
 
 # Color Tokens
 
-Color tokens make up a big part of Tokens Studio for Figma, as they're so tightly integrated with Figma's styles but give you options that Styles don't offer (yet).
+Color tokens play a significant role in Tokens Studio for Figma because they are tightly integrated with Figma's styles. They provide additional options that Figma's native styles currently don't offer.
+With color tokens you can:
 
-You can use color tokens for various things:
-
-- Updating color styles
-- Creating color styles
-- Applying a fill color
-- Applying a stroke color
+- Apply a fill color
+- Apply a stroke color
+- Create color styles
+- Update color styles
 
 ## Solid colors
 
-There are multiple ways you can write color tokens:
+By default, a token references a Solid Paint (single color).
+There are a few ways you can define color tokens:
 
 - Hex: `#ff0000`
 - RGB: `rgb(255, 0, 0)`
 - RGBA: `rgba(255, 0, 0, 1)`
 - HSLA: `hsla(120, 50%, 50%, 1)`
 
-By default, a token references a Solid Paint (single colors).
-
 ## Modifiers (Pro)
 
-We also support color modifiers which can help you create dynamic colors based on some parameters, such as lighten, darken, mix and alpha. You can [read more about modifiers here](/tokens/color-modifiers).
+We also support color modifiers which can help you create dynamic colors based on some parameters such as lighten, darken, mix and alpha. You can [read more about modifiers here](/tokens/color-modifiers).
 
 ## Gradients
 
-You can define gradient color tokens by specifying a css-like syntax: `linear-gradient(45deg, #ffffff 0%, #000000 100%)`.
+You can define gradient color tokens by specifying a css-like syntax: `linear-gradient (45deg, #87CEEB 0%, #008000 100%)`.
+<img src='/linear-gradient.png' alt="Creating a linear gradient - step 1" width="800"/>
 
-If you need multiple color steps, just add more (minimum is 2). `linear-gradient(45deg, #ffffff 0%, #ff0000 50%, #000000 100%)`
+If you need multiple color steps, just add more (minimum is 2). `linear-gradient(45deg, #0000FF 0%, #87CEEB 50%, #008000 100%)`
+<img src='/linear-gradient-x3.png' alt="Multiple values in gradients - step 2" width="800"/>
 
 ### Token References
-The great thing about token gradients is that you can create gradients that reference another token, something that is not possible in Figma itself. `linear-gradient(45deg, $colors.white 0%, $colors.black 100%)`
+The beauty of token gradients is that you can create gradients that reference another token, which is not possible within Figma itself. For example, you can define a gradient using token references by using the `{`, like this: linear-gradient (45deg, {colors.yellow.400} 0%, {colors.blue.700} 100%).
```

Comment:
This is causing a build issue, wrap the  linear-gradient (45deg, {colors.yellow.400} 0%, {colors.blue.700} 100%). with '`' or you'll need to escape the `{` `}` characters